ex02.ex

```diff
@@ -3,62 +3,35 @@ Code.load_file "support.ex"
 
 defmodule Ex02 do
   use ExUnit.Case
-  import Support
 
-  ##############################################################################
-  # 2: 3 questions,  15 points available
-  ##############################################################################
-
-  # These exercises manipulate lists using anonymous functions.
-  # Replace the text
-  #
-  #      your_anonymous_function(...)
-  #
-  # with your solution. Note that even though the placeholder
-  # takes just one line, your solution may take several.
-  #
-  # You can test your answers by running:
-  #
-  #     elixir ex02.ex
-
-
-  ##################
-  # 2.1:  5 points #
   ########################################################################
   # Write an anonymous function that takes two numbers and returns a two #
   # element list. The first element of the list will be the sum of the   #
   # numbers, and second should be the difference                         #
   ########################################################################
 
-  list2a = your_anonymous_function(1, 2)
-
+  list2a = fn (x, y) -> [x + y, x - y] end
   assert list2a.(1, 2)    == [ 3, -1 ]
   assert list2a.(-1, 100) == [ 99, -101 ]
 
-  ##################
-  # 2.2:  5 points #
   ##################################
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
-
+  list2b = &([&1 + &2, &1 - &2])
```

Comment:
No grade deduction, buit you can also leave off the parens here, so the idiomatic way would be to write

~~~ elixir
list2b = &[ &1 + &2, &1 - &2 ]
~~~